src/com/facebook/buck/rules/KnownBuildRuleTypes.java

```diff
@@ -670,6 +673,16 @@ static Builder createBuilder(
             defaultCxxPlatform,
             testTempDirOverride));
     builder.register(new KeystoreDescription());
+    KotlinBuckConfig kotlinBuckConfig = new KotlinBuckConfig(config);
```

Comment:
I think these are usually declared somewhere above (this "block" is just full of `builder.register`)